jbpm-human-task/jbpm-human-task-audit/src/test/java/org/jbpm/services/task/audit/service/TaskAuditBaseTest.java

```diff
@@ -319,8 +319,10 @@ private void testDescriptionUpdate(String oldDescription, String newDescription,
         List<TaskEvent> taskEvents = taskAuditService.getAllTaskEvents(taskId, new QueryFilter());
         if (changeExpected) {
             Assertions.assertThat(taskEvents).hasSize(2);
-            Assertions.assertThat(taskEvents.get(1).getMessage()).contains(String.valueOf(oldDescription),
-                    String.valueOf(newDescription));
+            assertEquals(getUpdateFieldLogMessage("Description",
```

Comment:
@nmirasch just a minor comment here, I think would be better if in the expected value we had the whole String instead of using an auxiliary method to build it. It would make the test easier to read as well, wdyt?